src/ListActions/ListActions.js

```diff
@@ -484,7 +484,9 @@ var ListActions = module.exports = kind(
 		var listAction, i;
 
 		this.listActionComponents = [];
-		this.$.listActions.destroyClientControls();
+		if (this.generated) {
```

Comment:
There was no real bug here, just a fairly minor optimization for create time that I had forgotten to update when reviewing a previous PR. The guard isn't necessarily for the existence of client controls, but more that we we don't need to do this at create time.